### PR TITLE
Update Number ranges. Add MEC.

### PR DESCRIPTION
**Pull request by thurahtetaung**
Updated Ooredoo, Telenor, Mytel number ranges,
Added MEC as separate operator.
Added Unit test for MEC
All tests passed.

### CODE REVIEW

File: myanmar-phonenumber.js [hunk 1]
```diff
@@ -125,7 +130,7 @@
         phoneNumber = this.normalizeInput(phoneNumber);
         if (operatorRe.ooredoo.test(phoneNumber) || operatorRe.telenor.test(phoneNumber) || operatorRe.mytel.test(phoneNumber)) {
           networkType = this.NETWORK_TYPE.GSM;
-        } else if (operatorRe.mpt.test(phoneNumber)) {
+        } else if (operatorRe.mpt.test(phoneNumber) || operatorRe.mec.test(phoneNumber)) {
```
Comment (kaungmyatlwin) [inline myanmar-phonenumber.js:133]: Shed me a light on this. MEC doesn't have its own dedicated network type?

**Review by kaungmyatlwin — APPROVED** [review]
Overall looks good to me. Please have a look at [this comment](https://github.com/kaungmyatlwin/myanmar-phonenumber/pull/10/files#r1069102831) then we're good to go!

File: myanmar-phonenumber.js (hunk 1 above)
Comment (thurahtetaung) [reply myanmar-phonenumber.js:133]: MEC only have CDMA 800 type and the regex for it is already in MPT's if case so I just added an OR. We can also do a separate else if case for MEC with only CDMA regex if the former approach is confusing.

**Review by thurahtetaung — COMMENTED** [review]
MEC only have CDMA 800 type and the regex for it is already in MPT's if case so I just added an OR. We can also do a separate else if case for MEC with only CDMA regex if the former approach is confusing.

**Comment by kaungmyatlwin** [comment]
Thanks for your contribution @thurahtetaung ! Release will be available soon! :)

**Comment by kaungmyatlwin** [comment]
@thurahtetaung Your change is released in https://www.npmjs.com/package/myanmar-phonenumber/v/0.3.3.

**Pull request merged**